netbox/circuits/graphql/filters.py

```diff
@@ -0,0 +1,50 @@
+import strawberry
+import strawberry_django
+from circuits import filtersets, models
+
+from netbox.graphql.filter_mixins import autotype_decorator, BaseFilterMixin
+
+__all__ = (
```

Comment:
Should we look into automating the generation of these filters? Or would you prefer to leave it explicit for now to avoid potentially complicating things?

Reply:
I think leave it explicit for now as you may want to possibly add something to the filter beyond the auto-generated response.